### PR TITLE
Nwchem update to 7.0.0

### PR DESCRIPTION
**Pull request by garvct**
-Updated install script to 7.0.0, added openblas.
-moved run_h2o_freq.sh to a generic run_nwchem.pbs script.
-Updated readme.

### CODE REVIEW

File: apps/nwchem/readme.md [hunk 1]
```diff
@@ -36,12 +34,12 @@ azhpc-connect -u hpcuser headnode
 ## Run the NWCHEM h2o_freq scenario
 To run on a single node with 4 cores run
 ```
-qsub -l select=1:ncpus=60:mpiprocs=4 apps/nwchem/run_h2o_freq.sh
+qsub -l select=1:ncpus=120:mpiprocs=4 -v INPUTDIR=/data/nwchm run_nwchem.sh
```
Comment (xpillons) [inline apps/nwchem/readme.md:37]: wrong extension, should be .pbs I guess

File: apps/nwchem/readme.md [hunk 2]
```diff
@@ -36,12 +34,12 @@ azhpc-connect -u hpcuser headnode
...
 ```
-qsub -l select=1:ncpus=60:mpiprocs=4 apps/nwchem/run_h2o_freq.sh
+qsub -l select=1:ncpus=120:mpiprocs=4 -v INPUTDIR=/data/nwchm run_nwchem.sh
 ```
 
-To run on two HB nodes with 8 total cores (4 cores on each node) run
+To run on two HBv2 nodes with 8 total cores (4 cores on each node) run
 ```
-qsub -l select=2:ncpus=60:mpiprocs=4 apps/nwchem/run_h2o_freq.sh
+qsub -l select=2:ncpus=120:mpiprocs=4 -v INPUTDIR=/data/nwchm run_nwchem.sh
```
Comment (xpillons) [inline apps/nwchem/readme.md:42]: wrong extension, should be .pbs I guess

**Review by xpillons — APPROVED** [review]
good for me. need to update the readme to fix the wrong extension to be .pbs instead of .sh

File: apps/nwchem/readme.md (hunk 2 above)
Comment (garvct) [reply apps/nwchem/readme.md]: Thanks, corrected the readme file.

**Pull request merged**